modules/programs/firefox.nix

```diff
@@ -6,13 +6,35 @@ let
 
   cfg = config.programs.firefox;
 
+  mozillaConfigPath =
+    if pkgs.stdenv.hostPlatform.isDarwin then
```

Comment:
Typically the format for `if`-expressions with short `then` and `else` branches would be
```
mozillaConfigPath =
  if pkgs.stdenv.hostPlatform.isDarwin
  then "Library/Application Support/Mozilla"
  else ".mozilla";
```

you could also add

```
inherit (pkgs.stdenv.hostPlatform) isDarwin;
```
above so that it becomes
```
mozillaConfigPath =
  if isDarwin
  then "Library/Application Support/Mozilla"
  else ".mozilla";
```

Reply:
Ah, much nicer 👌 I've updated all the if branches based on `isDarwin` in 12b7f23